docs/ml-classification-regression.md

```diff
@@ -302,6 +302,38 @@ Refer to the [Java API docs](api/java/org/apache/spark/ml/classification/OneVsRe
 </div>
 </div>
 
+## Naive Bayes
+
+[Naive Bayes](http://en.wikipedia.org/wiki/Naive_Bayes_classifier) are a family of simple 
+probabilistic classifiers based on applying Bayes' theorem with strong (naive) independence 
+assumptions between the features. More information about the spark.ml implementation can be 
+found further in the section on [Naive Bayes in MLlib](mllib-naive-bayes.html#naive-bayes-sparkmllib).
+
```

Comment:
Thanks for taking a look. The wiki link already provided a good overall introduction to Naive Bayes. I'll add some clarification, although in the mllib documents, it clarifies naive Bayes supports both Multinomial and Bernoulli.